hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/JetProperties.java

```diff
@@ -88,6 +88,16 @@
     public static final HazelcastProperty JET_HOME
             = new HazelcastProperty("jet.home", "");
 
+    /**
+     * If {@code false}, Jet will check during initialization that IMDG module has compatible version.
```

Comment:
Please wrap this text (72 chars starting from the 1st text char), otherwise we can merge it.